openehr-rm/src/main/java/com/nedap/archie/rm/composition/Instruction.java

```diff
@@ -34,6 +33,7 @@
     @XmlElement(name = "wf_definition")
     private DvParsable wfDefinition;
     @Nullable
+    @XmlElement(name = "activities")
     private List<Activity> activities = new ArrayList<>();
```

Comment:
I would expect this to be the default. Any specific reason for adding xmlelement here?